Example/SmileID/ProductCell.swift

```diff
@@ -1,30 +1,41 @@
-import SwiftUI
 import SmileID
+import SwiftUI
 
 struct ProductCell: View {
```

Comment:
I don't think we should modify ProductCell at all, in this case. This is tying the network state and this independent ui component together too tightly. 

Note that this component takes `onClick` as a parameter. In HomeView, you can pass in something for onClick so that we show a toast when they click on a product but there is not internet.

We should launch the product even when there is no internet as long as the toast is displayed. For 2 reasons
1. So we can continue to test network error behavior as devs
2. More importantly, we're adding offline mode. In which case, we won't want to block the products anwyays